src/test/datascience/helpers.ts

```diff
@@ -21,7 +21,7 @@ export function defaultDataScienceSettings(): IJupyterSettings {
             level: 'off'
         },
         experiments: {
-            enabled: false,
+            enabled: true,
```

Comment:
This should be off,
on CI, everythign should be predicatable.
We don't want a feature turning on one CI and not the other.